.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           - macos-latest
           - ubuntu-latest
           - windows-latest
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11-dev"]
```

Comment:
You shouldn't set a prerelease as required. Let the CI be happy if 3.11-dev fails. See https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

Also, drop 3.6.

Reply:
Right!